alf/bin/train_play_test.py

```diff
@@ -318,8 +322,8 @@ def test_ac_breakout(self):
 
     def test_ac_cart_pole(self):
         def _test_func(returns, lengths):
-            self.assertGreater(np.mean(returns[-2:]), 198)
-            self.assertGreater(np.mean(lengths[-2:]), 198)
+            self.assertGreater(returns[-1], 195)
```

Comment:
instead of assertGreater, just assert last 5 values == (....)?

This way, any unintended change in training behavior will be detected by the test.

Reply:
Discussed with Wei about this, we probably don't want a very strict test case.